src/pages/FrontendMentor/ElvaYBPractice2/styles/SliderComponent.styled.ts

```diff
@@ -0,0 +1,288 @@
+import styled from 'styled-components'
+import { Slider, Switch } from 'antd'
+
+import BackgroundPic from '../../../../assets/ElvaYongbin/BackgroundPic.svg'
+import PatternCirclePic from '../../../../assets/ElvaYongbin/images/pattern-circles.svg'
+import SliderIcon from '../../../../assets/ElvaYongbin/images/icon-slider.svg'
```

Comment:
can consider removing some of the components here. as this is for slider component's styling, there shouldn't be styling for other components.

Reply:
This is before extraction.